src/wasm/wasm-s-parser.cpp

```diff
@@ -881,6 +878,42 @@ Type SExpressionWasmBuilder::stringToType(const char* str,
   throw ParseException(std::string("invalid wasm type: ") + str);
 }
 
+HeapType SExpressionWasmBuilder::stringToHeapType(const char* str,
+                                                  bool allowError,
```

Comment:
It doesn't look like `allowError` is used. Do we need it?

Reply:
Added this analogous to `stringToType` which also has `allowError` and doesn't use it, and wasn't sure what to do about it. Do you prefer to remove it at both places or just here?

Reply:
Huh, strange. Might as well remove it in both places, I guess.